sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -94,6 +96,41 @@ FormRecognizerClient formRecognizerClient = new FormRecognizerClientBuilder()
 credential.update("{new_key}");
 ```
 
+#### Create a Form Recognizer client using Microsoft identity platform (formerly Azure Active Directory)
+
+Azure SDK for Java supports an Azure Identity package, making it easy to get credentials from Microsoft identity
+platform. First, add the package:
+
+[//]: # ({x-version-update-start;com.azure:azure-identity;current})
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
```

Comment:
if we do need this section, could we point to the Azure Identity readme instead of saying here ho to add the package?
Another concern is that we are hard coding the version. Is that intentional? Only that version works with FR? how do we keep this version updated?

Reply:
In Java, we have a script running to update the version.

Reply:
Yes. We talked it in the Team Java Channel:  https://teams.microsoft.com/l/message/19:5e673e41085f4a7eaaf20823b85b2b53@thread.skype/1589413307528?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1589413307528&teamName=Azure%20SDK&channelName=Language%20-%20Java&createdTime=1589413307528